pkg/translate/translate.go

```diff
@@ -119,9 +119,9 @@ var (
 
 				"Telemetry.Components.Namespace": {"global.telemetryNamespace", nil},
 
-				"Security.ControlPlaneMtls.Value":    {"global.controlPlaneSecurityEnabled", nil},
-				"Security.DataPlaneMtlsStrict.Value": {"global.mtls.enabled", nil},
-				"Security.Components.Namespace":      {"global.securityNamespace", nil},
+				"Security.ControlPlaneMtls":     {"global.controlPlaneSecurityEnabled", nil},
```

Comment:
This may work for reverse translation but I think it breaks the forward translation - since the unmarshaled value is a BoolValue rather than bool.

Reply:
yep, looks like I have to add an override for reverse case